proto-bindings/build.sbt

```diff
@@ -44,28 +46,56 @@ generateCsharpBindings := {
   new File(s"cpg-proto-bindings.${dotnetVersion}.nupkg")
 }
 
+lazy val installProtoc = taskKey[File]("downloads configured protoc version for your platform, unless already available")
+installProtoc := {
+  import sys.process._
+  import better.files.FileExtensions
+  val protocVersion = (ProtobufConfig/version).value
+  val protocBinary = new File(protocBinaryPath)
+  val isAlreadyInstalled = protocBinary.exists && s"$protocBinaryPath --version".!!.contains(protocVersion)
+
+  if (!isAlreadyInstalled) {
+    val platform= (System.getProperty("os.name"), System.getProperty("os.arch")) match {
+      case ("Linux", "amd64") => "linux-x86_64"
+      case (name, "amd64") if name.startsWith("Windows") => "win64"
+      case (name, arch) if name.toLowerCase.contains("mac") && arch.contains("64") => "osx-x86_64"
```

Comment:
is it `mac` or `darwin`?

Reply:
no idea, this is what i found on the interwebz